### PR TITLE
INTEGRATIONS-0 Add CircleCI and AutoPackaging

### PR DESCRIPTION
**Pull request by bartboy011**
This PR adds CircleCI integration (currently a WIP) and AutoPackaging support. AutoPackaging allows a dev to create the full, uploadable package by running `composer run build`, rather than using the Magento UI. 

@ActiveCampaign/integration

### CODE REVIEW

**Comment by bartboy011** [comment]
Testing

**Comment by bartboy011** [comment]
Testing

**Comment by bartboy011** [comment]
Testing

**Comment by bartboy011** [comment]
@cristiangrama This is ready now!

File: bin/package-files [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```
Comment (cristiangrama) [inline bin/package-files]: @bartboy011 If we build automatically this file should change in the follow up PR.

**Pull request merged**